src/ui/common/src/components/integrations/dialogs/dialog.tsx

```diff
@@ -196,187 +228,193 @@ const IntegrationDialog: React.FC<Props> = ({
     </Box>
   );
 
-  let serviceDialog;
+  // TODO: Add service Dialog to the service list.
```

Comment:
clean this up / remove.

Figure out what's needed for the GCS Dialog first since it had some special logic there.